--- db-3.0.55/os/os_map.c.jbj	Sat Apr  1 16:13:49 2000
+++ db-3.0.55/os/os_map.c	Sat Apr  1 16:16:19 2000
@@ -33,6 +33,8 @@
 static int __os_map __P((DB_ENV *, char *, DB_FH *, size_t, int, int, void **));
 #endif
 
+int __os_shmget_perms = 0600;
+
 /*
  * __os_r_sysattach --
  *	Create/join a shared memory region.
@@ -68,7 +70,7 @@
 #endif
 #if defined(HAVE_SHMGET)
 		if (F_ISSET(infop, REGION_CREATE) &&
-		   (rp->segid = shmget(0, rp->size, IPC_PRIVATE | 0600)) == -1)
+		   (rp->segid = shmget(0, rp->size, IPC_PRIVATE | __os_shmget_perms)) == -1)
 			return (__os_get_errno());
 
 		if ((infop->addr = shmat(rp->segid, NULL, 0)) == (void *)-1) {
