diff -ur db-3.0.55.old/dist/Makefile.in db-3.0.55/dist/Makefile.in
--- db-3.0.55.old/dist/Makefile.in	Thu Nov 11 17:09:48 1999
+++ db-3.0.55/dist/Makefile.in	Sat Apr 15 12:15:07 2000
@@ -328,92 +328,93 @@
 	@test -f $(rm) || (echo 'rm not found.'; exit 1)
 
 install_include:
-	@echo "Installing DB include files: $(includedir) ..."
-	@test -d $(includedir) || \
-	    ($(mkdir) -p $(includedir) && $(chmod) $(dmode) $(includedir))
-	@cd $(includedir) && $(rm) -f db.h db_185.h db_cxx.h
-	@$(cp) -p db.h db_185.h $(srcdir)/include/db_cxx.h $(includedir)
-	@cd $(includedir) && $(chmod) $(fmode) db.h db_185.h db_cxx.h
+	@echo "Installing DB include files: $(DESTDIR)$(includedir) ..."
+	@test -d $(DESTDIR)$(includedir) || \
+	    ($(mkdir) -p $(DESTDIR)$(includedir) \
+	    && $(chmod) $(dmode) $(DESTDIR)$(includedir))
+	@cd $(DESTDIR)$(includedir) && $(rm) -f db.h db_185.h db_cxx.h
+	@$(cp) -p db.h db_185.h $(srcdir)/include/db_cxx.h $(DESTDIR)$(includedir)
+	@cd $(DESTDIR)$(includedir) && $(chmod) $(fmode) db.h db_185.h db_cxx.h
 
 install_static: install_setup
-	@echo "Installing DB library: $(libdir) ..."
-	@test -d $(libdir) || \
-	    ($(mkdir) -p $(libdir) && $(chmod) $(dmode) $(libdir))
-	@cd $(libdir) && $(rm) -f $(libdb)
-	@$(cp) -p $(libdb) $(libdir)
-	@cd $(libdir) && $(chmod) $(fmode) $(libdb)
+	@echo "Installing DB library: $(DESTDIR)$(libdir) ..."
+	@test -d $(DESTDIR)$(libdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(libdir) && $(chmod) $(dmode) $(DESTDIR)$(libdir))
+	@cd $(DESTDIR)$(libdir) && $(rm) -f $(libdb)
+	@$(cp) -p $(libdb) $(DESTDIR)$(libdir)
+	@cd $(DESTDIR)$(libdir) && $(chmod) $(fmode) $(libdb)
 
 install_static_cxx: install_setup
-	@echo "Installing DB C++ static library: $(libdir) ..."
-	@test -d $(libdir) || \
-	    ($(mkdir) -p $(libdir) && $(chmod) $(dmode) $(libdir))
-	@cd $(libdir) && $(rm) -f $(libcxx)
-	@$(cp) -p $(libcxx) $(libdir)
-	@cd $(libdir) && $(chmod) $(fmode) $(libcxx)
+	@echo "Installing DB C++ static library: $(DESTDIR)$(libdir) ..."
+	@test -d $(DESTDIR)$(libdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(libdir) && $(chmod) $(dmode) $(DESTDIR)$(libdir))
+	@cd $(DESTDIR)$(libdir) && $(rm) -f $(libcxx)
+	@$(cp) -p $(libcxx) $(DESTDIR)$(libdir)
+	@cd $(DESTDIR)$(libdir) && $(chmod) $(fmode) $(libcxx)
 
 install_dynamic: install_setup
-	@echo "Installing DB library: $(libdir) ..."
-	@test -d $(libdir) || \
-	    ($(mkdir) -p $(libdir) && $(chmod) $(dmode) $(libdir))
-	@cd $(libdir) && \
+	@echo "Installing DB library: $(DESTDIR)$(libdir) ..."
+	@test -d $(DESTDIR)$(libdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(libdir) && $(chmod) $(dmode) $(DESTDIR)$(libdir))
+	@cd $(DESTDIR)$(libdir) && \
 	    $(rm) -f $(libso) $(libso_default) $(libso_major)
-	@$(INSTALLER) $(libso_target) $(libdir)
-	@cd $(libdir) && $(ln) -s $(libso) $(libso_default)
-	@cd $(libdir) && $(ln) -s $(libso) $(libso_major)
-	@$(LIBTOOL) --mode=finish $(libdir)
+	@$(INSTALLER) $(libso_target) $(DESTDIR)$(libdir)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libso) $(libso_default)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libso) $(libso_major)
+	@$(LIBTOOL) --mode=finish $(DESTDIR)$(libdir)
 
 install_dynamic_cxx: install_setup
-	@echo "Installing DB C++ library: $(libdir) ..."
-	@test -d $(libdir) || \
-	    ($(mkdir) -p $(libdir) && $(chmod) $(dmode) $(libdir))
-	@cd $(libdir) && \
+	@echo "Installing DB C++ library: $(DESTDIR)$(libdir) ..."
+	@test -d $(DESTDIR)$(libdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(libdir) && $(chmod) $(dmode) $(DESTDIR)$(libdir))
+	@cd $(DESTDIR)$(libdir) && \
 	    $(rm) -f $(libxso) $(libxso_default) $(libxso_major)
-	@$(INSTALLER) $(libxso_target) $(libdir)
-	@cd $(libdir) && $(ln) -s $(libxso) $(libxso_default)
-	@cd $(libdir) && $(ln) -s $(libxso) $(libxso_major)
+	@$(INSTALLER) $(libxso_target) $(DESTDIR)$(libdir)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libxso) $(libxso_default)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libxso) $(libxso_major)
 
 install_tcl: install_setup
-	@echo "Installing DB Tcl library: $(libdir) ..."
-	@test -d $(libdir) || \
-	    ($(mkdir) -p $(libdir) && $(chmod) $(dmode) $(libdir))
-	@cd $(libdir) && \
+	@echo "Installing DB Tcl library: $(DESTDIR)$(libdir) ..."
+	@test -d $(DESTDIR)$(libdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(libdir) && $(chmod) $(dmode) $(DESTDIR)$(libdir))
+	@cd $(DESTDIR)$(libdir) && \
 	    $(rm) -f $(libtso) $(libtso_default) $(libtso_major)
-	@$(INSTALLER) $(libtso_target) $(libdir)
-	@cd $(libdir) && $(ln) -s $(libtso) $(libtso_default)
-	@cd $(libdir) && $(ln) -s $(libtso) $(libtso_major)
+	@$(INSTALLER) $(libtso_target) $(DESTDIR)$(libdir)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libtso) $(libtso_default)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libtso) $(libtso_major)
 
 install_java: install_setup
-	@echo "Installing DB Java library: $(libdir) ..."
-	@test -d $(libdir) || \
-	    ($(mkdir) -p $(libdir) && $(chmod) $(dmode) $(libdir))
-	@cd $(libdir) && \
+	@echo "Installing DB Java library: $(DESTDIR)$(libdir) ..."
+	@test -d $(DESTDIR)$(libdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(libdir) && $(chmod) $(dmode) $(DESTDIR)$(libdir))
+	@cd $(DESTDIR)$(libdir) && \
 	    $(rm) -f $(libjso) $(libjso_default) $(libjso_major)
-	@$(INSTALLER) $(libjso_target) $(libdir)
-	@cd $(libdir) && $(ln) -s $(libjso) $(libjso_default)
-	@cd $(libdir) && $(ln) -s $(libjso) $(libjso_major)
+	@$(INSTALLER) $(libjso_target) $(DESTDIR)$(libdir)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libjso) $(libjso_default)
+	@cd $(DESTDIR)$(libdir) && $(ln) -s $(libjso) $(libjso_major)
 
 install_docs: install_setup
-	@echo "Installing documentation: $(docdir) ..."
-	@test -d $(docdir) || \
-	    ($(mkdir) -p $(docdir) && $(chmod) $(dmode) $(docdir))
-	@cd $(docdir) && $(rm) -rf *
-	@cd $(srcdir)/docs && $(cp) -pr * $(docdir)/
+	@echo "Installing documentation: $(DESTDIR)$(docdir) ..."
+	@test -d $(DESTDIR)$(docdir) || \
+	    ($(mkdir) -p $(DESTDIR)$(docdir) && $(chmod) $(dmode) $(DESTDIR)$(docdir))
+	@cd $(DESTDIR)$(docdir) && $(rm) -rf *
+	@cd $(srcdir)/docs && $(cp) -pr * $(DESTDIR)$(docdir)/
 
 install_utilities: install_setup
-	@echo "Installing DB utilities: $(bindir) ..."
-	@test -d $(bindir) || \
-	    ($(mkdir) -p $(bindir) && $(chmod) $(dmode) $(bindir))
-	@cd $(bindir) && $(rm) -f $(PROGS)
-	@$(INSTALLER) -fp $(PROGS) $(bindir)
-	@cd $(bindir) && (test ! -f $(strip) || $(strip) $(PROGS))
-	@cd $(bindir) && $(chmod) $(emode) $(PROGS)
+	@echo "Installing DB utilities: $(DESTDIR)$(bindir) ..."
+	@test -d $(DESTDIR)$(bindir) || \
+	    ($(mkdir) -p $(DESTDIR)$(bindir) && $(chmod) $(dmode) $(DESTDIR)$(bindir))
+	@cd $(DESTDIR)$(bindir) && $(rm) -f $(PROGS)
+	@$(INSTALLER) -fp $(PROGS) $(DESTDIR)$(bindir)
+	@cd $(DESTDIR)$(bindir) && (test ! -f $(strip) || $(strip) $(PROGS))
+	@cd $(DESTDIR)$(bindir) && $(chmod) $(emode) $(PROGS)
 
 uninstall:
-	-cd $(bindir) && $(rm) -f $(PROGS)
-	-cd $(includedir) && $(rm) -f db.h db_185.h db_cxx.h
-	-cd $(libdir) && $(rm) -f $(libdb) $(libcxx) $(libso_target) $(libso) \
+	-cd $(DESTDIR)$(bindir) && $(rm) -f $(PROGS)
+	-cd $(DESTDIR)$(includedir) && $(rm) -f db.h db_185.h db_cxx.h
+	-cd $(DESTDIR)$(libdir) && $(rm) -f $(libdb) $(libcxx) $(libso_target) $(libso) \
 	    $(libjso) $(libjso_target) $(libtso) $(libtso_target)
-	-cd $(docdir) && $(rm) -rf *
+	-cd $(DESTDIR)$(docdir) && $(rm) -rf *
 
 ##################################################
 # Object build rules.
