--- ./dist/Makefile.in.org	Sat Feb 14 19:09:23 2004
+++ ./dist/Makefile.in	Sat Feb 14 19:12:44 2004
@@ -60,7 +60,7 @@
 LIBS=		@LIBS@
 LIBSO_LIBS=	@LIBSO_LIBS@
 
-libdb=		libdb.a
+libdb=		libdb-$(SOVERSION).a
 libso_base=	libdb
 libso=		$(libso_base)-$(SOVERSION).@SOSUFFIX@
 libso_static=	$(libso_base)-$(SOVERSION).a
@@ -135,8 +135,8 @@
 # local libraries, for example.  Do that by adding -I options to the DB185INC
 # line, and -l options to the DB185LIB line.
 ##################################################
-DB185INC=	-c @CFLAGS@ -I$(srcdir)/include @CPPFLAGS@
-DB185LIB=
+DB185INC=	-c @CFLAGS@ -I/usr/include/db1 -I$(srcdir)/include @CPPFLAGS@
+DB185LIB=	-static -ldb1
 
 ##################################################
 # NOTHING BELOW THIS LINE SHOULD EVER NEED TO BE MODIFIED.
@@ -247,6 +247,7 @@
 # for Embedix.  Note: "all" must be the first target in the Makefile.
 ##################################################
 all: @BUILD_TARGET@
+static:	@DEFAULT_LIB@ @ADDITIONAL_LIBS@ @ADDITIONAL_LANG@
 
 install: @INSTALL_TARGET@
 
